config/deployment.yml

```diff
@@ -43,19 +43,25 @@ spec:
         volumeMounts:
         - name: template-fs
           mountPath: /etc/kappctrl-mem-tmp
-        securityContext:
-          runAsUser: 1000
-          runAsGroup: 2000
+        - name: home
+          mountPath: /home/kapp-controller
```

Comment:
Make this a volume so it is created with writeable permissions for whatever user we're running as